src/views/v2/list/event/featured-image.php

```diff
@@ -9,30 +9,28 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.3
+ * @version TBD
  *
  */
-$event    = $this->get( 'event' );
-$event_id = $event->ID;
 
-if ( ! has_post_thumbnail( $event_id ) ) {
+if ( empty( $event->thumbnail->large->url ) ) {
 	return;
 }
 
 ?>
 <div class="tribe-events-calendar-list__event-featured-image-wrapper tribe-common-g-col">
 	<a
-		href="<?php echo esc_url( tribe_get_event_link( $event_id ) ); ?>"
-		title="<?php echo esc_attr( get_the_title( $event_id ) ); ?>"
+		href="<?php echo esc_url( $event->permalink ) ?>"
+		title="<?php echo esc_attr( $event->post_title ) ?>"
```

Comment:
Use the `get_the_title` template tag to have the filtered version of the event title:
```suggestion
		title="<?php echo esc_attr( get_the_title( $event->ID ) ) ?>"
```